src/onemod/stage/model_stage/kreg_stage.py

```diff
@@ -0,0 +1,46 @@
+"""Kreg stage."""
+
+import fire
+
+from onemod.config.model_config import KregConfig
+from onemod.stage import ModelStage
+
+
+class KregStage(ModelStage):
```

Comment:
The way these stage classes are written means we can immediately write simple tests for the construction and running of a dummy Pipeline 👍 which is not a bad place to get things started I think

Reply:
I think it is a good place to start :-)